app/models/manageiq/providers/vmware/cloud_manager/vm/operations.rb

```diff
@@ -1,3 +1,14 @@
 module ManageIQ::Providers::Vmware::CloudManager::Vm::Operations
+  extend ActiveSupport::Concern
+
   include_concern 'Power'
+
+  def raw_destroy
+    raise "VM has no #{ui_lookup(:table => "ext_management_systems")}, unable to destroy VM" unless ext_management_system
+    with_provider_object(&:undeploy)
+    ext_management_system.with_provider_connection do |service|
+      service.delete_vapp(ems_ref)
+    end
+    update_attributes!(:raw_power_state => "DELETED")
```

Comment:
Is `DELETED` a valid power state?  How about "off" since it is a valid [vcloud_director state](https://github.com/xlab-si/fog-vcloud-director/blob/master/lib/fog/vcloud_director/parsers/compute/vm.rb#L70) and also recognized by the [vcloud provider](https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/cloud_manager/vm.rb#L9-L15)